Omise/Models/Models.cs

```diff
@@ -489,6 +521,44 @@ public partial class Dispute : ModelBase {
         }
     }
 
+    public partial class Document : ModelBase {
+        [JsonProperty("download_uri")]
+        public string DownloadURI { get; set; }
+        [JsonProperty("filename")]
+        public string filename { get; set; }
```

Comment:
Should it be capitalize the first letter?

Reply:
Good catch! Thank you.